src/main/java/com/minelittlepony/bigpony/mod/mixin/MixinEntityPlayer.java

```diff
@@ -7,13 +7,15 @@
 import org.spongepowered.asm.mixin.Implements;
 import org.spongepowered.asm.mixin.Interface;
 import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.Shadow;
 import org.spongepowered.asm.mixin.injection.*;
 import org.spongepowered.asm.mixin.injection.callback.CallbackInfoReturnable;
 
 @Mixin(EntityPlayer.class)
 @Implements(@Interface(iface = IEntityPlayer.class, prefix = "bigpony$"))
 public abstract class MixinEntityPlayer extends EntityLivingBase {
 
+    public float heightFactor = 1F;
```

Comment:
This really has no reason to be in the mixin. The only reason `eyeHeight` is here is because forge adds it.

Reply:
Actually, Let me do some thinking on this.